canonical-web-and-design/maas-ui
# Update general form

## Done
- Refactor general form into the containers view to match the same pattern as the rest of the forms
- Remove configuration component as not needed
- Redirected `/configuration` to `/configuration/general`
- Add action button component

## QA
- Go to `/configuration/general`
- See that form still works as expected
- See that the action button is used on the form
